3ds Max/Max2Babylon/Exporter/BabylonExporter.Material.Pbr.cs

```diff
@@ -0,0 +1,435 @@
+﻿using System.Linq;
+using Autodesk.Max;
+using BabylonExport.Entities;
+
+namespace Max2Babylon
+{
+    /// <summary>
+    /// We are building decorator to homogenize and simplify the access to Material properties. 
+    /// </summary>
+    public class PbrGameMaterialDecorator : MaterialDecorator
+    {
+        private static readonly float[] Point3Default = new float[] { 0, 0, 0 };
+
+        protected static ITexmap _getTexMap(IIGameMaterial materialNode, string name)
```

Comment:
It will move to Material decorator class at the next PR. This is a function i inherit from historical code. This is protected because it serve the hierarchy into property getter such
`public virtual ITexmap BaseColorMap => _getTexMap(_node, "base_color_map");`
we can make it non static if we want a statefull version (see next question)

Reply:
Sounds like a plan. I'll leave this comment open for now